macros/schema.yml

```diff
@@ -2,16 +2,48 @@ version: 2
 
 macros:
   - name: channel_classification
-    description: A macro used to perform channel classifications
-
+    description: '{{ doc("macro_channel_classification") }}'
   - name: channel_spend
-    description: A macro for the user to overwrite it with a sql script to extract total ad spend by channel
-
+    description: '{{ doc("macro_channel_spend") }}'
   - name: conversion_clause
-    description: A macro to specify how to filter on conversion events
-
+    description: '{{ doc("macro_conversion_clause") }}'
   - name: conversion_value
-    description: A macro that specifies either a single column or a calculated value that represents the value associated with the conversion
-
+    description: '{{ doc("macro_conversion_value") }}'
+  - name: get_lookback_date_limits
+    description: '{{ doc("macro_get_lookback_date_limits") }}'
+    arguments:
+      - name: limit_type
+        type: string
+        description: Can be either 'min' or 'max' depending on if the upper or lower boundary date needs to be returned
   - name: create_udfs
-    description: A macro which creates user defined functions that will be used in the models
+    description: '{{ doc("macro_create_udfs") }}'
+  - name: path_transformation
+    description: '{{ doc("macro_path_transformation") }}'
+    arguments:
+      - name: transformation_type
+        type: string
+        description: >
+          A type of transformation that needs to be executed E.g. 'unique_path'.
+          Needs to be one of the following: exposure_path, first_path, remove_if_not_all, unique_path
+          More details here https://docs.snowplow.io/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-fractribution-data-model/#path-transform-options
```

Comment:
The next item on my to-do is to add the docs, noticed we were missing some. Thanks for spotting this, of course I copied. this from the only wrong line that has been changed since. will add them here, too.